dbt_platform_helper/commands/internal.py

```diff
@@ -21,3 +27,93 @@ def migrate_service_manifests():
         service_manager.migrate_copilot_manifests()
     except PlatformException as error:
         click_io.abort_with_error(str(error))
+
+
+@internal.group(cls=ClickDocOptGroup)
+def service():
+    """Subgroup for 'internal service' commands."""
+
+
+@service.command(help="Trigger an ECS deployment.")
+@click.option("--name", required=True, help="The name of the ECS service to create or update.")
+@click.option(
+    "--environment",
```

Comment:
```suggestion
    "--env",
```

Can we use env for the CLI option for consistency with other commands?